core/client/app/controllers/setup/two.js

```diff
@@ -3,6 +3,7 @@ import {request as ajax} from 'ic-ajax';
 import ValidationEngine from 'ghost/mixins/validation-engine';
 
 export default Ember.Controller.extend(ValidationEngine, {
+    needs: 'setup',
```

Comment:
`needs` is deprecated.  Controllers can be injected via `Ember.inject.controller()`.